roles/network_plugin/weave/tasks/main.yml

```diff
@@ -15,8 +15,3 @@
   template:
     src: weave-net.yml.j2
     dest: "{{ kube_config_dir }}/weave-net.yml"
-
-- name: Weave | Fix nodePort for Weave
```

Comment:
Maybe this task should remain with a when condition based on the version and you should add another task that effectively removes the file when version is 2.5.0 (especially in case of versions upgrades)